### ISSUE TITLE
Error loading Youtube feed

### ISSUE THREAD

**Issue by pluja** [pluja/Parasitter#44]
```
Traceback (most recent call last):
  File "/home/ubuntu/Yotter/venv/lib/python3.8/site-packages/feedparser.py", line 398, in __getattr__
    return self.__getitem__(key)
  File "/home/ubuntu/Yotter/venv/lib/python3.8/site-packages/feedparser.py", line 356, in __getitem__
    return dict.__getitem__(self, key)
KeyError: 'published_parsed'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/ubuntu/Yotter/venv/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/ubuntu/Yotter/venv/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/ubuntu/Yotter/venv/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/ubuntu/Yotter/venv/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/ubuntu/Yotter/venv/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/ubuntu/Yotter/venv/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/ubuntu/Yotter/venv/lib/python3.8/site-packages/flask_login/utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "/home/ubuntu/Yotter/app/routes.py", line 199, in youtube
    videos = getYoutubePosts(ids)
  File "/home/ubuntu/Yotter/app/routes.py", line 745, in getYoutubePosts
    time = datetime.datetime.now() - datetime.datetime(*vid.published_parsed[:6])
  File "/home/ubuntu/Yotter/venv/lib/python3.8/site-packages/feedparser.py", line 400, in __getattr__
    raise AttributeError("object has no attribute '%s'" % key)
AttributeError: object has no attribute 'published_parsed'
```
Sometimes Youtube feed throws this error.

**Comment by eetukarvonen** [comment]
``` [2020-09-09 14:34:32,413] ERROR in app: Exception on /youtube [GET]
Traceback (most recent call last):
  File "/home/mint/Documents/Projects/Yotter/venv/lib/python3.8/site-packages/flask/app.py", line 2447, in wsgi_app
    response = self.full_dispatch_request()
  File "/home/mint/Documents/Projects/Yotter/venv/lib/python3.8/site-packages/flask/app.py", line 1952, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/home/mint/Documents/Projects/Yotter/venv/lib/python3.8/site-packages/flask/app.py", line 1821, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/home/mint/Documents/Projects/Yotter/venv/lib/python3.8/site-packages/flask/_compat.py", line 39, in reraise
    raise value
  File "/home/mint/Documents/Projects/Yotter/venv/lib/python3.8/site-packages/flask/app.py", line 1950, in full_dispatch_request
    rv = self.dispatch_request()
  File "/home/mint/Documents/Projects/Yotter/venv/lib/python3.8/site-packages/flask/app.py", line 1936, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/home/mint/Documents/Projects/Yotter/venv/lib/python3.8/site-packages/flask_login/utils.py", line 272, in decorated_view
    return func(*args, **kwargs)
  File "/home/mint/Documents/Projects/Yotter/app/routes.py", line 205, in youtube
    videos.sort(key=lambda x: x.date, reverse=True)
TypeError: '<' not supported between instances of 'datetime.datetime' and 'time.struct_time'
```

Seems like sometimes video.date is 'datetime' and sometimes 'time' and sort function gives TypeError.

**Comment by pluja** [comment]
@eetukarvonen are you still having this error? I think I fixed it on f6e70fdbe38bc71d027dd7e74ee0aac1a415d4a9

**Comment by eetukarvonen** [comment]
Looks like it is working now.
